R/apply.scaling.R

```diff
@@ -1,15 +1,14 @@
 apply.scaling <- function(data.matrices, scaling.factors) {
 
-	# data.matrices can be a single matrix or a list of matrices
-	# if the data is a single matrix then the class will be 'matrix'	
-	if (class(data.matrices)[1] == 'matrix') {
-	
+	# data.matrices can be a single matrix/data frame or a list of matrices/data frames
+	# if the data is a single matrix / data.frame, then the class will be 'matrix' or 'data.frame'
```

Comment:
**(suggestion)** I think this sentence needs to be clarified. I think the data means the `data.matrices`, right?